h/templates/layouts/base.html.jinja2

```diff
@@ -66,7 +66,8 @@
         {
           "raven": {
             "dsn": "{{ request.sentry.get_public_dsn('https') }}",
-            "release": "{{ h_version }}"
+            "release": "{{ h_version }}",
+            "userid": {{ request.authenticated_userid | to_json }}
```

Comment:
It would seem more consistent to do `"{{ request.authenticated_userid }}"` here instead of using `to_json`.